DashAI/front/src/components/ExperimentsTable.jsx

```diff
@@ -128,7 +128,7 @@ function ExperimentsTable({ handleNewExperiment }) {
             key="delete-button"
             icon={<DeleteIcon />}
             label="Delete"
-            onClick={handleDeleteExperiment(params.id)}
+            // TODO: onClick={handleDeleteExperiment(params.id)}
```

Comment:
This is destroying the database. On click handlers receive a function, not an evaluated function